src/BugTracker/project.json

```diff
@@ -6,13 +6,14 @@
     "version": "1.0.0-*",
     "webroot": "wwwroot",
     "dependencies": {
-        "Kestrel": "1.0.0-*",
-        "Microsoft.AspNet.Server.IIS": "1.0.0-*",
-        "Microsoft.AspNet.Mvc": "6.0.0-*",
-        "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
-        "Microsoft.AspNet.Diagnostics": "1.0.0-*",
-        "Microsoft.AspNet.StaticFiles": "1.0.0-*",
-        "Microsoft.AspNet.SignalR.Server": "3.0.0-*"
+        "Kestrel": "1.0.0-beta4-*",
```

Comment:
We don't know the final beta4 build version yet. The wildcard will be swept once they're decided.